client/src/components/RequestDash/requestTestData.js

```diff
@@ -0,0 +1,48 @@
+const errands = [
```

Comment:
Nice. I'll have to borrow this.

Do we have a group test data folder? might not be a bad idea to put them all together so everyone has access.